.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
According to the requirements, the test.md file which is currently appearing in your project structure should have been ignored here, consider listing it here as you have done with the node_modules file in order to fulfil the requirements